modules/openapi-generator/src/main/resources/JavaSpring/beanValidationPathParams.mustache

```diff
@@ -1 +1 @@
-{{! PathParam is always required, no @NotNull necessary }}{{>beanValidationCore}}
\ No newline at end of file
+{{! PathParam is always required, no @NotNull necessary }}{{^useOptional}}{{>beanValidationCore}}{{/useOptional}}{{#useOptional}}{{#required}}{{>beanValidationCore}}{{/required}}{{/useOptional}}
```

Comment:
path parameter is always required. Does it need the `{{#required}} ... {{/required}}` check?

Reply:
Good catch thanks. Revert it